### DISCUSSION TITLE
In new updates, we're looking forward to this ...

### DISCUSSION THREAD

**Discussion by Ambassador-of-programming** [flet-dev/flet#2670] [General]
Good day!
Me and my team that develops useful things for people using Flet. We were waiting for the possibility to install our own libraries from requirements.txt by default, which are available in python, but under the hood they use other languages and during the build we get errors because of this.
It is extremely inconvenient to use different kinds of additional manipulations every time to add your own libraries that are not written in pure python.
Should I expect a solution to this problem in the new release?

**Comment by FeodorFitsner** [comment]
Yes, we are going to solve that problem in the next releases. It's #1 priority item in our TODO list.
